### PR TITLE
Feat: Update font

### PR DESCRIPTION
**Pull request by truph01**
### Fixed Issues

$ https://github.com/Expensify/App/issues/43512
PROPOSAL: https://github.com/Expensify/App/issues/43512#issuecomment-2162284229

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/170341821/b8cb0f1d-0cf1-4285-a7e2-5b09a90fb9a7

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/170341821/0695e6b5-2181-49f7-87e6-9bcdc9110c35

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/170341821/764a4a0f-5988-48c9-af42-6f2d1357223d

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/170341821/9e27069f-85ba-4c68-9739-af52e1853a91

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

- Large screen:

https://github.com/Expensify/App/assets/170341821/127c8695-f8ee-4a9c-bfe6-0de5e2cd6792

- Small screen:

https://github.com/Expensify/App/assets/170341821/11245905-56a2-4e7f-a87c-7b9d6eef78c8

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/170341821/5b00eeea-6134-416a-b369-3603f4bb4bc4

</details>

### CODE REVIEW

File: src/styles/index.ts
Comment (shawnborton) [inline src/styles/index.ts:410]: We should have existing styles for this, no?

File: src/styles/index.ts
Comment (shawnborton) [inline src/styles/index.ts:455]: Is this a new style?

**Comment by shawnborton** [comment]
On desktop, the To & From should not be in bold.

**Comment by shawnborton** [comment]
Can you check the spacing here:
CleanShot 2024-06-14 at 08 21 45@2x

We just want everything there to have an 8px gap

File: src/styles/index.ts
Comment (truph01) [reply src/styles/index.ts]: Oh. There was a similar style named textNormal:
```
        textNormal: {
            fontSize: variables.fontSizeNormal,
        },
```
So I removed ```fontSizeNormal``` and use ```textNormal``` instead.

File: src/styles/index.ts
Comment (truph01) [reply src/styles/index.ts:410]: Yes. We do not have this style before.

**Comment by truph01** [comment]
@shawnborton Reply your https://github.com/Expensify/App/pull/43733#issuecomment-2167293132:
- Yes. The gap between avatar and name is 8px:
image

**Comment by shawnborton** [comment]
> Yes. The gap between avatar and name is 8px:

Ah sorry, that comment was from the mobile view of the Search page.

**Comment by shawnborton** [comment]
Can you make it so that these To/From columns on desktop vertically center everything? Right now it looks like it's aligned to the top and it feels a bit off:
CleanShot 2024-06-14 at 18 53 13@2x

**Comment by shawnborton** [comment]
We may need to make the date column slightly wider as a result of this. We currently have a date PR in review that also adds the ability to see the year in dates for expenses from prior years, so perhaps as soon as that is merged, we can adjust the column widths there.

**Comment by shawnborton** [comment]
This isn't your fault but it's something I think we should fix: the small text in mobile view under the merchant has some weird alignment going on, I think in part because we are giving it a min-height of 20px:
CleanShot 2024-06-14 at 19 00 45@2x

Can we make it so that we just use the standard line-height of the text, which should be 14px since it's micro-sized font:
CleanShot 2024-06-14 at 19 02 20@2x

And then we just vertically center everything? Basically removing the min-height in my first screenshot of this comment should fix that.

**Review by luacmartins — COMMENTED** [review]
@truph01 please let me know once you're addressed Shawn's comments

**Comment by truph01** [comment]
@luacmartins Sure. I am working on it. It is related to style fix, so I need to test on all platforms.

**Comment by luacmartins** [comment]
Sounds good, thanks for the update!

**Comment by truph01** [comment]
@shawnborton 

> Can you make it so that these To/From columns on desktop vertically center everything? Right now it looks like it's aligned to the top and it feels a bit off:

- Currently, I do not have any solution for it. The parent's height is 21, and the child's height is 16, so the ```alignItem: 'center'``` or ```'alignSelf: 'center'``` do not work.

> We may need to make the date column slightly wider as a result of this. We currently have a date PR in review that also adds the ability to see the year in dates for expenses from prior years, so perhaps as soon as that is merged, we can adjust the column widths there.

- We already have a logic to handle it. If we display the year, the date column is grow up. This is that change: https://github.com/Expensify/App/pull/43413/files#diff-097e3d461183958f84757dc8be87e3f24b862654235a9b3f59f75ae74a738e1


> The small text in mobile view under the merchant has some weird alignment going on

- I fixed it by removing the min line height attribute:

image

**Comment by c3024** [comment]
@truph01 Could you add `Tests` for all changes made for both large and small screens?

**Review by c3024 — CHANGES_REQUESTED** [review]

**Comment by c3024** [comment]
@shawnborton 

This font size for headers appears too small to me on Chrome on a 27 inch monitor.

Screenshot 2024-06-18 at 7 39 23 PM

Is this okay?

**Comment by shawnborton** [comment]
Yeah, we don't want to change the font size for the column headers at all.

**Comment by shawnborton** [comment]
> We already have a logic to handle it. 

I mean even when there is no year present, the column width was too skinny and it was cutting off the simple date. We will need to make it wider.

> Currently, I do not have any solution for it. The parent's height is 21, and the child's height is 16

To be clear, all text in the desktop view should now have a line height of our normal line height (it's 21 now but it's being fixed to go back to 20). So can we just fix that alignment but making sure we aren't setting 16 as the line height in the To/From column, or any other column for that matter?

**Comment by shawnborton** [comment]
Just tested again, notice how even when no year is applied, you still get a cut off date column:
CleanShot 2024-06-19 at 09 20 07@2x

So we need to slightly increase that width.

**Comment by shawnborton** [comment]
For my comment about the vertical alignment of the avatar + to/from name, maybe we don't worry about that for now because I think when we reduce our line height back to 20px, it will look a bit better?
CleanShot 2024-06-19 at 09 22 41@2x

**Comment by shawnborton** [comment]
Or we could simply add a 1px top margin to the avatar wrapper, which does seem to balance it out:
CleanShot 2024-06-19 at 09 24 47@2x

**Comment by shawnborton** [comment]
On mobile, seeing some weird alignment when there is a category and tag present:
CleanShot 2024-06-19 at 09 25 32@2x

**Comment by shawnborton** [comment]
Looks like there is still 4px gap here on mobile, when we want it all to be just 8px:
CleanShot 2024-06-19 at 09 32 10@2x

**Comment by truph01** [comment]
> Just tested again, notice how even when no year is applied, you still get a cut off date column:

- I updated. Here is result:

image

> On mobile, seeing some weird alignment when there is a category and tag present

- I updated. Here is result:

image

> Also while we are here, can we make it so that the icon + date is vertically centered too?

- I updated. Here is result:

image


> Looks like there is still 4px gap here on mobile, when we want it all to be just 8px:

- I updated. Here is result:

image

**Comment by shawnborton** [comment]
Lovely. Just a few more comments. 

On mobile, can we make it so that the smaller text uses the same baseline alignment?
CleanShot 2024-06-19 at 10 51 26@2x

Ideally the merchant and amount use the same font size and line height, and then both of those should have the same gap between them and the smaller text below.

Then a different comment - given that we are losing a lot of horizontal space with the updated font size, could we update the expense type column on desktop to be 20px wide? Basically the same exact width as the icon itself. We need to update that in the column header as well.

**Comment by truph01** [comment]
> Ideally the merchant and amount use the same font size and line height, and then both of those should have the same gap between them and the smaller text below.

@shawnborton Currently, the merchant has lineHeight: 21 and the amount has lineHeight: 20. So should I update the merchant's lineHeight to 20?

**Comment by shawnborton** [comment]
You could make it so that they both use 20, yup.

**Comment by truph01** [comment]
@shawnborton I addressed your [comment](https://github.com/Expensify/App/pull/43733#issuecomment-2178130278)

**Comment by shawnborton** [comment]
Nice! Going to re-build.

File: src/styles/utils/index.ts
Comment (shawnborton) [inline src/styles/utils/index.ts:1562]: FWIW, 80 isn't quite big enough when we have a year present:
CleanShot 2024-06-19 at 16 40 39@2x

And for the other case, instead of 50, can you try using 52 or 56?

**Review by luacmartins — COMMENTED** [review]
@truph01 we're getting close! we have one more request from @shawnborton though

**Comment by truph01** [comment]
@shawnborton I resolved your [comment](https://github.com/Expensify/App/pull/43733#discussion_r1646328296)

**Comment by shawnborton** [comment]
Sweet, let me rebuild again.

**Review by luacmartins — CHANGES_REQUESTED** [review]
The total column shown for report transactions has a smaller font:

Screenshot 2024-06-20 at 2 13 43 PM

**Comment by shawnborton** [comment]
Good catch!

Also wanted to bump @Expensify/design to check this out - I'm wondering if the lack of bold letters for the To/From actually looks way worse on mobile?
CleanShot 2024-06-20 at 22 34 32@2x

**Comment by truph01** [comment]
> The total column shown for report transactions has a smaller font:

- @luacmartins I fixed your comment. This is result:
image

**Comment by truph01** [comment]
Just need to wait for the @Expensify/design's response in https://github.com/Expensify/App/pull/43733#issuecomment-2181512414

**Comment by shawnborton** [comment]
Cool, let's wait until they chime in today.

**Comment by dubielzyk-expensify** [comment]
> Also wanted to bump @Expensify/design to check this out - I'm wondering if the lack of bold letters for the To/From actually looks way worse on mobile?

I don't feel super strongly here so happy to lean on you both. Unbolded saves a few precious pixels, but bolded makes it stand out a bit more against the rest of the regular text. I probably lean slightly bold, but again, not super strongly.

**Comment by shawnborton** [comment]
If we put them back to bold, we get something like this:
CleanShot 2024-06-21 at 08 13 19@2x

Which I think I like better, as it has some harmony with the button or badge on the right side which uses bold weight as well.

**Comment by shawnborton** [comment]
Another small thing I am noticing: it looks like the expense cards are only 100px tall:
CleanShot 2024-06-21 at 08 14 30@2x

Whereas in Figma, we have them at 108:
CleanShot 2024-06-21 at 08 15 08@2x

I think that's because this top header wrapper here only has a margin-bottom of 8px:
CleanShot 2024-06-21 at 08 15 44@2x

Whereas it should actually have a margin bottom of 16px:
CleanShot 2024-06-21 at 08 16 40@2x


Can you make that change as well please? Thanks!

**Comment by truph01** [comment]
> Whereas it should actually have a margin bottom of 16px:

- @shawnborton I fixed it. This is result:
Screenshot 2024-06-21 at 16 30 02

> Also wanted to bump @Expensify/design to check this out - I'm wondering if the lack of bold letters for the To/From actually looks way worse on mobile?

- What is the final decision on this?

**Comment by shawnborton** [comment]
I think we should do bold names, so let's implement that quickly and then test again. Thanks!

**Comment by truph01** [comment]
> I think we should do bold names, so let's implement that quickly and then test again. Thanks!

- Here is the result, the font-weight is 700:
Screenshot 2024-06-21 at 16 58 46

**Comment by shawnborton** [comment]
Looking good to me! Let's get this into final review.

**Comment by c3024** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

**Review by luacmartins — APPROVED** [review]
LGTM

**Comment by luacmartins** [comment]
@c3024 @shawnborton all yours

**Review by c3024 — APPROVED** [review]
LGTM too!

**Comment by dannymcclain** [comment]
Sorry meant to weigh in on this before, but I'm into the bold for names on mobile!

**Review by luacmartins — APPROVED** [review]

**Comment by luacmartins** [comment]
Chatted 1:1 with @shawnborton and we're good to merge!

**Comment by shawnborton** [comment]
Sorry for the late feedback, but @truph01 I found a case where the date column isn't wide enough and causes truncation:
CleanShot 2024-06-24 at 11 18 18@2x

Can you follow up with a quick fix for that?

**Comment by truph01** [comment]
@shawnborton Sure. I wonder which value should we use?https://github.com/Expensify/App/blob/2026f62b5074d892163f837ca7b030b84c8e350f/src/styles/utils/index.ts#L1562

**Comment by shawnborton** [comment]
I think just bumping to 88 might do the trick, but let's check a variety of dates to make sure. Thanks @truph01 !

**Comment by luacmartins** [comment]
It seems like we need at least 92px. The date `May 30, 2030` or `May 30, 2000` gets truncated otherwise.

**Comment by luacmartins** [comment]
PR [here](https://github.com/Expensify/App/pull/44292)

**Pull request merged**
